SPECS/polkit/polkit.spec

```diff
@@ -101,39 +96,37 @@ fi
 %{_datarootdir}/dbus-1/system-services/org.freedesktop.PolicyKit1.service
 %{_datarootdir}/locale/*
 %{_datarootdir}/polkit-1/actions/*.policy
-%{_sysconfdir}/dbus-1/system.d/org.freedesktop.PolicyKit1.conf
+%{_datadir}/dbus-1/system.d/org.freedesktop.PolicyKit1.conf
 %{_sysconfdir}/pam.d/polkit-1
 %{_sysconfdir}/polkit-1/rules.d/50-default.rules
+%{_datadir}/gettext/its
 
 %files devel
 %defattr(-,root,root)
 %{_includedir}/%{name}-1/
 %{_libdir}/lib%{name}-*.a
 %{_libdir}/lib%{name}-*.so
 %{_libdir}/pkgconfig/*.pc
-%{_datadir}/gettext/its/polkit.its
-%{_datadir}/gettext/its/polkit.loc
 
 %changelog
-*   Thu Jun 03 2021 Andrew Phelps <anphel@microsoft.com> - 0.116-5
--   Enable check tests (with exception of unsupported "polkitbackend" tests)
+*   Wed Sep 22 2021 Jon Slobodzian <joslobo@microsoft.com> 0.119-1
+-   Initial CBL-Mariner import from Photon (license: Apache2)
+-   Bump to polkit 0.119
 
-*   Thu Jun 03 2021 Jon Slobodzian <josloboe@microsoft.com> - 0.116-4
```

Comment:
We usually leave the changelog untouched even for version bumps. After all for CBL-Mariner the previous versions had a slightly different history.